### PR TITLE
Add PIH to example apps

### PR DESCRIPTION
**Pull request by eljhkrr**
Fixes https://github.com/medic/config-pih/issues/492

### CODE REVIEW

File: content/en/apps/examples/pih.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "YendaNafe CHT app by PIH in Malawi"
+linkTitle: "PIH Malawi"
+weight: 
+description: >
+  The YendaNafe app co-designed by PIH Malawi and Medic
+keywords:  
+---
+
+Since 2017, PIH Malawi and Medic have collaboratively developed the CHT-based YendaNafe app, a digital health application for community-based health service provision and coordination. Now, in the spirit of radical collaboration and openness, PIH and Medic have coordinated the release of the full application source code of YendaNafe as first of its kind “Integrated CHT App”, free for all to access and replicate.
```
Comment (abbyad) [inline content/en/apps/examples/pih.md:10]: Would be good to spell out PIH as Partners In Health at least once on the page, either in the description or first paragraph, and include an appropriate link, like pih.org or pih.org/country/malawi

Comment (eljhkrr) [reply content/en/apps/examples/pih.md:10]: Updated

File: content/en/apps/examples/pih.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This is a monumental milestone in the digital health space and for the CHT Community. We look forward to learning how you will leverage these resources and welcome your feedback and continued innovation.
+
+## Resources to Get Started
+
+Here are a few additional resources to help get you started with the maternal and newborn health reference application.
+
+- View the [configuration code for this reference app](https://github.com/medic/cht-pih-malawi-app)
+- Install the reference app following these [easy installation instructions]({{< ref "apps/tutorials/local-setup" >}})
+- Modify the maternal and newborn reference application for your project context using [configuration best practices]({{< ref "design/best-practices" >}})
```
[the remark this answers is not in the document]
Comment (eljhkrr) [reply content/en/apps/examples/pih.md]: Updated

**Comment by abbyad** [comment]
The updates look good @eljhkrr! I'll let @nsitaula complete their review so that it can be approved.

**Review by eljhkrr — COMMENTED** [review]
CHT docs site has better support for column format as seen in [ANC docs](https://docs.communityhealthtoolkit.org/apps/examples/anc/) so we can improve on the use of images

**Comment by nsitaula** [comment]
Hi @mrjones-plip could you please review PIH documentation? i.e. content -> en -> apps-> examples
cc @antonykhaemba

**Comment by mrjones-plip** [comment]
@n-orlowski - I have a favor to ask: can you please attach same sized images for the icons used in this PR?   Specifically, we need to break out the icons at the top of the full size images as seen highlighted in red here:

image

So that we turn them into these 3 PNGs, which are identically sized and have a transparent background: [1](http://localhost:1313/apps/examples/anc/delivery-3.png), [2](http://localhost:1313/apps/examples/anc/delivery-2.png), [3](http://localhost:1313/apps/examples/anc/pregnancy-3.png).  

Any ones that already exist (like from [this page](https://docs.communityhealthtoolkit.org/apps/examples/anc/#workflows)) just re-use!   Here's the full list of images so you can easily click through to come up with a unique list of icons. There's 11 unique flow images,  but likely only 8 (or less!) unique icons as there's a lot of re-use:

* [Human_Immunodeficiency_Virus.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/Human_Immunodeficiency_Virus.png)
* [IMCI_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/IMCI_workflow.png)
* [TB_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/TB_workflow.png)
* [TB_workflow_screened.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/TB_workflow_screened.png)
* [delivery_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/delivery_workflow.png)
* [family_planning_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/family_planning_workflow.png)
* [malnutrition_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/malnutrition_workflow.png)
* [ncd.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/ncd.png)
* [ncd_1.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/ncd_1.png)
* [post_natal_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/post_natal_workflow.png)
* [pregnancy_workflow.png](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/pregnancy_workflow.png)

Attach a zip to prevent GH from monkeying resizing etc.

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
@antonykhaemba  and @nsitaula - looks good! Nice work.

The big change is we need to convert all the workflow images from [this](https://raw.githubusercontent.com/medic/cht-docs/fc600ec49c4999e1bf22e9098bb25b577dd90f53/content/en/apps/examples/pih/pregnancy_workflow.png) to be native markdown.  Unlike last time, we should use the `workflow-table` [as seen here.](https://docs.communityhealthtoolkit.org/apps/examples/anc/#pregnancy-workflow)  Here is the markdown to render this example (below).

I've already [asked for](https://github.com/medic/cht-docs/pull/771#issuecomment-1307971325) hi-res, non-fuzzy and evenly sized transparent background images.  However, you don't need to wait and can concurrently work on converting all the images to text and learn how to use the `workflow-table` feature.  Let me know if you need help!


```markdown
{{< workflow-table
  condition-image="pregnancy-1.png"
  task-image="pregnancy-2.png"
  resolution-image="pregnancy-3.png"
 >}}
  {{% workflow
        condition="Upon discovering a pregnancy, a CHW submits a [Pregnancy Form](https://docs.google.com/drawings/d/1u4OQIgTyzUysFv9Cop-C54nrbPiuPmq7GTpBTFmwZn0/edit) confirming a new pregnancy with the estimated gestational age."
        task="At the 8 ANC touchpoints defined by the WHO, the CHW receives a [Pregnancy Home Visit Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.wf9x0zhfeasi) to let her know that it’s time to check in on the pregnant woman."
        resolution="CHW submits [Pregnancy Home Visit Form](https://docs.google.com/drawings/d/1_2i6XTMtMkrfQ6NFNjcEDwJPS8i0rEeQlaPgYQhacSw/edit), demonstrating that she provided ANC counseling, gathered information from prior facility visits, and screened for danger signs."
  %}}
  {{% workflow
        condition="A [Pregnancy Form](https://docs.google.com/drawings/d/1u4OQIgTyzUysFv9Cop-C54nrbPiuPmq7GTpBTFmwZn0/edit) is submitted with the gestational age unknown."
        task="A [Pregnancy Visit Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.2pls723gu6wl) appears every 2 weeks for 42 weeks."
        resolution="CHW submits [Pregnancy Home Visit Form](https://docs.google.com/drawings/d/1_2i6XTMtMkrfQ6NFNjcEDwJPS8i0rEeQlaPgYQhacSw/edit). If the gestational age is entered, the workflow will change to [Pregnancy Home Visit Tasks](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.wf9x0zhfeasi)."
  %}}
  {{% workflow
        condition="A CHW submits a [Pregnancy Home Visit Form](https://docs.google.com/drawings/d/1_2i6XTMtMkrfQ6NFNjcEDwJPS8i0rEeQlaPgYQhacSw/edit) that includes an upcoming facility visit date."
        task="A [Health Facility ANC Reminder Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.v3b7bata6j) 1 week ahead of the facility visit to remind the woman to attend."
        resolution="CHW submits *Health Facility ANC Reminder Form*, confirming that she called or visited the woman to remind her of her upcoming facility visit."
  %}}
  {{% workflow
        condition="If the CHW notices danger signs at any time, then she submits a *Danger Sign Form* and immediately refers the patient to the facility."
        task="A [Danger Sign Follow-Up Task](https://docs.google.com/document/d/17pJXBf2gEB2wD1P5g9S5XQ2u4QnLGMiUU4sX0XQsvSk/edit#heading=h.82ea7ww1x1k) will appear immediately and is due 3 days later. Tasks persists for 7 days after due date."
        resolution="CHW submits a *Danger Sign Follow-Up Form*, verifying that she called or visited the woman to confirm that she attended the facility. If this is not received, another *Danger Sign Follow-Up Task* is triggered."
  %}}
{{< /workflow-table >}}
```

**Comment by n-orlowski** [comment]
[images here!](https://drive.google.com/drive/u/0/folders/1dozsWRZyotdTFDugOc6a90wuPHe8Pn1e)

**Comment by mrjones-plip** [comment]
You rock - thanks @n-orlowski !!

**Comment by nsitaula** [comment]
Hi @mrjones-plip, I have updated the document, could you please review?

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
These new `workflow-table`s look GREAT -  Thanks @nsitaula !

Two more changes please:
1. Remove unused images now that we're using the `workflow-table`
2. convert the [remaining image](https://raw.githubusercontent.com/medic/cht-docs/d21cc7100ad879157306bbc244e0ec98e92804e5/content/en/apps/examples/pih/users_hierarchy.png) to a markdown table with text.

**Comment by nsitaula** [comment]
Hi @mrjones-plip , I have made the changes. Could you please review and let me know if I missed anything?

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
@niraj21  - it doesn't look like you addressed my two main concerns.  Please give it another go!

The changes I requested last time are:

* Remove unused images now that we're using the workflow-table
* convert the [remaining image](https://raw.githubusercontent.com/medic/cht-docs/d21cc7100ad879157306bbc244e0ec98e92804e5/content/en/apps/examples/pih/users_hierarchy.png) to a markdown table with text.

**Comment by nsitaula** [comment]
Hi @mrjones-plip , sorry for the confusion. Instead of committing directly to the branch, I created a  [new branch](https://github.com/medic/cht-docs/blob/nsitaula-patch-1/content/en/apps/examples/pih.md). I will make changes to the branch [pih_example_app](https://github.com/medic/cht-docs/tree/pih_example_app) and remove the new branch I created.

**Comment by nsitaula** [comment]
Hi @mrjones-plip, I have made the changes and removed the other branch I created. Please let me know if I missed anything. sorry for the confusion.

**Review by mrjones-plip — APPROVED** [review]
@niraj21  - the final product looks GREAT!  nice work.

**Pull request merged**